check_service_property.go

```diff
@@ -1,8 +1,9 @@
 package opslevel
 
 type ServicePropertyCheckFragment struct {
-	Property  ServicePropertyTypeEnum `graphql:"serviceProperty"`
-	Predicate *Predicate              `graphql:"propertyValuePredicate"`
+	Property           ServicePropertyTypeEnum `graphql:"serviceProperty"`        // The property of the service that the check will verify.
+	PropertyDefinition *IdentifierInput        `graphql:"propertyDefinition"`     // The secondary key of the property that the check will verify (e.g. the specific custom property).
```

Comment:
is this supposed to be an input or an output? in the API this is a `PropertyDefinition` object, not an ID Input.

Reply:
ahh good catch

Reply:
Fixed